### PR TITLE
Enable WGM=-1.

### PR DESCRIPTION
**Pull request by hcman2**
WGM=-1 is different from WGM=1.
Treat WGM=-1 as normal negative WGM cases.

### CODE REVIEW

**Comment by hcman2** [comment]
@nakajee Please help to review this change.

File: tensilelite/Tensile/KernelWriterAssembly.py [hunk 1]
```diff
@@ -1975,8 +1975,8 @@ def graWorkGroup(self, kernel):
           module.add(wgmLabelPositive)
         else:
           module.add(SAbsI32(dst=sgpr("WGM"), src=sgpr("WGM"), comment="abs(WGM)"))
-          module.add(SCmpLeI32(src0=sgpr("WGM"), src1=1, comment="WGM <= 1 ?"))
-          module.add(SCBranchSCC1(labelName=wgmLabel.getLabelName(), comment="branch if WGM <= 1"))
+          module.add(SCmpEQI32(src0=sgpr("WGM"), src1=0, comment="WGM = 0 ?"))
```
Comment (nakajee) [inline tensilelite/Tensile/KernelWriterAssembly.py]: I think this changes the behavior of WGM=1.
What you can do is 
- move this s_cmp and branch below before s_abs (line 1978-1979 above 1977)
- change the condition to >=0

Then, WGM=0,1 are same as before and only -1 is different.
We do not actually need WGM calculation for -1 (just swap WG0 and WG1 should be enough).
That is optional. At least, please make sure you did not make any change for WGM!=-1.

Comment (hcman2) [reply tensilelite/Tensile/KernelWriterAssembly.py]: Fixed

**Review by aazz44ss — APPROVED** [review]
LGTM

**Review by nakajee — APPROVED** [review]
Thanks for your update.
Looks good.

**Comment by hcman2** [comment]
[----------] Global test environment tear-down
[==========] 13059 tests from 13 test suites ran. (3039262 ms total)
[  PASSED  ] 13059 tests.
gfx90a hipblaslt-test passed.

**Comment by hcman2** [comment]
=========== 18 passed, 74 skipped, 32 warnings in 3975.51s (1:06:15) ===========
gfx90a tox test passed.

**Comment by hcman2** [comment]
========== 66 passed, 26 skipped, 38 warnings in 40277.20s (11:11:17) ==========
gfx94x local test passed

**Pull request merged**